main_app/admin.py

```diff
@@ -1,8 +1,11 @@
 from django.contrib import admin
-from .models import compute_resource_model,profile_model,create_host_model,operating_system_model
+from .models import (compute_resource_model,
+                     create_host_model,
```

Comment:
Wrong indentation. It is bad practise to push new commit for each change( amend the commit )
`git push origin testing --amend`

Reply:
ok we will add all changes in a single commit